src/ui/views/ui_app.jade

```diff
@@ -125,7 +125,7 @@ block content
           err.style['text-align'] = 'center';
           err.style['background'] = '#F44336';
           err.style['padding'] = '25px';
-          err.innerText = '#{i18n('UI-WELCOME_ERROR')}';
+          err.innerText = '#{i18n_singleQuoteVar('UI-WELCOME_ERROR')}';
 
```

Comment:
I tried this and it works as expected without any other cahnges:
err.innerText = "#{i18n('UI-WELCOME_ERROR')}";